app/scripts/account-import-strategies/index.js

```diff
@@ -34,9 +34,7 @@ const accountImporter = {
       let wallet
       try {
         wallet = importers.fromEtherWallet(input, password)
-      } catch (e) {
-        console.log('Attempt to import as EtherWallet format failed, trying V3...')
-      }
+      } catch (e) {}
```

Comment:
Hmm - generally empty catch blocks are a bad sign 🤔 

Would it be better to move the contents of the `if (!wallet)` block below in this catch block instead? It does appear that `fromEtherWallet` should always return a truthy result unless an error is thrown, so it should be the equivalent of this except with more "obvious" error handling.